### PR TITLE
Add uwp specific requirements for SDP usage

### PR DESCRIPTION
**Pull request by IrinVoso**
This PR converts uwp specific metadata tags for SDP usage.

Tasks: 164896, 163110

### CODE REVIEW

File: ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs [hunk 1]
```diff
@@ -158,9 +158,97 @@ private T InitWithBasicProperties<T>(ReflectionItem item) where T : ItemSDPModel
                 rval.IsDeprecated = true;
             }
 
+            GenerateUWPRequirements(rval, item);
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs]: >GenerateUWPRequirements(rval, item); [](start = 12, length = 36)

wrap it with `if (_store.UWPMode)`?

File: ECMA2Yaml/ECMAHelper/Models/Constants.cs [hunk 2]
```diff
@@ -32,19 +32,33 @@ public class OPSMetadata
...
-            {"requirement_os_min_versions", MetadataDataType.StringArray },
-            {"deviceFamilies", MetadataDataType.StringArray },
-            {"deviceFamiliesVersions", MetadataDataType.StringArray },
-            {"apiContracts", MetadataDataType.StringArray },
-            {"apiContractsVersions", MetadataDataType.StringArray },
-            {"capabilities", MetadataDataType.StringArray },
-            {"xamlMemberSyntax", MetadataDataType.String },
-            {"contentSourcePath", MetadataDataType.String }
+            {SDKRequirementsName, MetadataDataType.String },
+            {SDKRequirementsUrl, MetadataDataType.String },
```
Comment (wh-alice) [inline ECMA2Yaml/ECMAHelper/Models/Constants.cs]: Order by Alphabet?

File: ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs (hunk 1 above)
Comment (IrinVoso) [reply ECMA2Yaml/ECMAHelper/YamlConverters/SDPYamlConverter.cs]: Done.

File: ECMA2Yaml/ECMAHelper/Models/Constants.cs (hunk 2 above)
Comment (IrinVoso) [reply ECMA2Yaml/ECMAHelper/Models/Constants.cs]: Done

**Review by TianqiZhang — APPROVED** [review]

**Pull request merged**